docs/faq.md

```diff
@@ -266,6 +267,93 @@ module.exports = {
 }
 ```
 
+## How to use Vite for the backend?
+
+Using TypeORM in a Vite project is pretty straight forward. However, when you use migrations, you will run into "...migration name is wrong. Migration class name should have a
+JavaScript timestamp appended." errors when running the production build.
+On production builds, files are optimized by default which includes mangling your code in order to minimize file sizes.
+
+You have 3 options to mitigate this. The 3 options are shown belown as diff to this basic "vite.config.ts"
+
+```ts
+import legacy from '@vitejs/plugin-legacy';
+import vue from '@vitejs/plugin-vue';
+import path from 'path';
+import { defineConfig } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    sourcemap: true,
+  },
+  plugins: [vue(), legacy()],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src'),
+    },
+  },
+});
+```
+
+### Option 1: Disable minify
+
+This is the most crude option and will result in significant larger files. Add `build.minify = false` to your config.
+
+```diff
+--- basic vite.config.ts
++++ disable minify vite.config.ts
+@@ -7,6 +7,7 @@
+ export default defineConfig({
+   build: {
+     sourcemap: true,
++    minify: false,
+   },
+   plugins: [vue(), legacy()],
+   resolve: {
+```
+
+### Option 2: Disable esbuild minify identifiers
+
+Vite uses esbuild as default minifier. You can disable mangling of identifiers by adding `esbuild.minifyIdentifiers = false` to your config.
+This will result in smaller file sizes, depending on your code base you will get diminishing returns as all identifiers will be kept at full lengt.
```

Comment:
```suggestion
Vite uses esbuild as the default minifier. You can disable mangling of identifiers by adding `esbuild.minifyIdentifiers = false` to your config.
This will result in smaller file sizes, but depending on your code base you will get diminishing returns as all identifiers will be kept at full length.
```